Enable SCSS files in the css

felipecrp opened the pull request:
This PR fix #243.

I changed the regexp to enable `css` and `scss` files.

Best regards,
Felipe

felipecrp commented:
@yihui Apparently, the error is related to the build process instead of the PR change itself. Can you help me fix it?

cderv commented on R/paged.R:
```diff
@@ -148,7 +148,7 @@ html_format = function(
   }
 
-  css2 = grep('[.]css$', css, value = TRUE, invert = TRUE)
+  css2 = grep('[.]s?css$', css, value = TRUE, invert = TRUE)
```
To be complete, the regex would be 
```suggestion
  css2 = grep('[.](?:sa|sc|c)ss$', css, value = TRUE, invert = TRUE)
```

felipecrp replied:
Ok. I will test the changes and update the docs.

cderv reviewed (changes requested):
Thanks. 

Following support in `rmarkdown::html_document()` we indeed did not bring support up to **pagedown**. I think we can do that. Thanks for catching that!

In addition of the regex, it would require to 

1. update the `css` argument description on the `html_paged` documentation in the roxygen comment
2. Add a NEWS bullet
3. Add a mention of SASS support in the main documentation (`inst/examples/index.Rmd`) under `## The CSS overriding mechanism` part.

yihui commented:
@felipecrp We can ignore the Travis error in this case. Thanks!

felipecrp commented:
@cderv and @yihui ,

Thank you for the support. 

I changed the regexp using the @cderv suggestion and tested the new release with `.scss` and `.sass` files. I also updated the roxygen doc, the documentation, and the NEWS.

I think this PR is ready for delivery.

Best regards,
Felipe

cderv reviewed (changes requested):
Thanks ! 

We still need to increase the min dependency of **rmarkdown** to 2.8 because of this feature. 
Currently it is 1.16
https://github.com/rstudio/pagedown/blob/bdc6a9285bc60e8b93756b30e919c10923088c72/DESCRIPTION#L23

This is the good practice, right @yihui ? 
I don't see the harm in doing that, all the more because it is always best to use the latest **rmarkdown** available.

cderv commented on NEWS.md:
```diff
@@ -8,6 +8,8 @@
 
 - In `chrome_print()`, set a timeout when R tries to detect headless Chrome.
 
+- Added support for Sass files.
```
```suggestion
- Added support for Sass files with `.scss` and `.sass` extension (thanks, @felipecrp, #244).
```

We usually thanks contributor in the NEWS item - so thank you ! :)

felipecrp replied:
You´re welcome 😄

cderv commented on DESCRIPTION:
```diff
@@ -20,7 +20,7 @@ Description: Use the paged media properties in CSS and the JavaScript
   running headers, etc. Applications of this package include books, letters,
   reports, papers, business cards, resumes, and posters.
-Imports: rmarkdown (>= 1.16), bookdown (>= 0.8), htmltools, jsonlite, later (>= 1.0.0),
+Imports: rmarkdown (>= 2.7), bookdown (>= 0.8), htmltools, jsonlite, later (>= 1.0.0),
```
```suggestion
Imports: rmarkdown (>= 2.8), bookdown (>= 0.8), htmltools, jsonlite, later (>= 1.0.0),
```

I thought we needed 2.8 (https://pkgs.rstudio.com/rmarkdown/news/index.html#rmarkdown-28). This is the one supporting the SASS files in `css`

yihui commented:
> This is the good practice, right @yihui ?

Yes, let's do that.

cderv reviewed (approved):
Thanks @felipecrp ! I'll merge this.

felipecrp commented:
Thank you for the support.
